src/RapidezStatamicServiceProvider.php

```diff
@@ -97,6 +103,9 @@ public function bootListeners() : self
                 Cache::forget('statamic-globals-' . Site::selected()->handle());
             });
 
+            Event::listen(NavCreated::class, SetCollectionsForNav::class);
```

Comment:
No, only when a new single navigation is created, this will set the allowed collections on that new single navigation. This are just defaults and when you need or feel like changing it you're free to do so.